main.go

```diff
@@ -118,7 +117,7 @@ func main() {
 		log.Fatal(err)
 	}
 
-	if tlsMinVersion != "" {
+	if tlsMinVersionValue.String() != "" {
```

Comment:
Or 
```suggestion
	if tlsMinVersionValue > 0 {
```

Reply:
`tlsMinVersionValue` is a variable of `tlsVersionValue` struct, I think this cannot be done.

Reply:
Could this one be changed as well as the other one below?

Reply:
I missed this, sure